### ISSUE TITLE
Quotation marks are not escaped when making dashboard

### ISSUE THREAD

**Issue by javierggt** [sot/skare3_tools#25]
As a result, one gets something like this:

```
pr_info: " 2020-05-02 - PR #52 Tweak print error (for accuracy and to not match "failed") ",
```

**Comment by jeanconn** [comment]
Oops.  Sorry about that in my commit message though at least there's a useful test case if you decide to fix the issue...

**Comment by javierggt** [comment]
No problem, the dashboard should be robust enough.

**Comment by javierggt** [comment]
Fixed in [4d3d58a](https://github.com/sot/skare3_tools/commit/4d3d58ac4ddf0983e5c806c3b4161dc4c0753da4). Jinja2 has autoescape, but that is for templates with html extension. For some reason, it did not work for tpl even if specified. Renamed the template.
